frontend/src/LanguageSwitcher.js

```diff
@@ -1,33 +1,41 @@
 import React from 'react'
 import { Query, Mutation } from 'react-apollo'
-import styled, { css } from 'react-emotion'
+import { css } from 'react-emotion'
 import {
   GET_LANGUAGE_QUERY,
   CHANGE_LANGUAGE_MUTATION,
 } from './utils/queriesAndMutations'
 
-const A = styled('a')`
+const buttonThatLooksLikeALink = css`
```

Comment:
I think we were / are just using default font sizes here, it's smaller than it used to be.
old:
![image](https://user-images.githubusercontent.com/8228248/50840928-996ea000-1331-11e9-857a-807b2c79e8f1.png)

new
![image](https://user-images.githubusercontent.com/8228248/50840940-a095ae00-1331-11e9-9636-9c4c8f0f5477.png)

Reply:
Good catch! Fixed!